doc/user/content/cloud/troubleshoot-cloud.md

```diff
@@ -37,11 +33,24 @@ that must be investigated and resolved.
 
 For more information, see [Monitoring: Logging](/ops/monitoring/#logging)
 
+## Disabling user indexes
+
+{{< warning >}}
+This feature is primarily meant for advanced administrators of Materialize.
+{{< /warning >}}
+
+If your Cloud deployment unexpectedly consumes all CPU or memory, you can troubleshoot by restarting with
+[indexes][api-indexes] on user-created objects disabled.  The option appears on the Deployment Details page.
+
+{{% troubleshooting/disable-user-indexes %}}
+
 ## Third-party monitoring tools
 
-Materialize supports integrations with [Datadog](/ops/monitoring/#datadog), [Grafana](/ops/monitoring/#grafana), and [Prometheus](/ops/monitoring/#prometheus).
+Materialize supports integrations with [Grafana](/ops/monitoring/#grafana) and [Prometheus](/ops/monitoring/#prometheus). For more information, see [Monitor Cloud].
 
 ## Related topics
 
-- [Monitoring](/ops/monitoring)
+- [Monitor Cloud]
```

Comment:
This link is broken. I suppose this is the one waiting on https://github.com/MaterializeInc/materialize/issues/8824